src/main/java/com/uid2/admin/store/writer/ScopedStoreWriter.java

```diff
@@ -59,7 +64,20 @@ public void upload(String data, JsonObject extraMeta) throws Exception {
         fileManager.uploadFile(location, dataFile, data);
         fileManager.uploadMetadata(metadata.getJson(), dataType, scope.getMetadataPath());
 
-        provider.loadContent(provider.getMetadata());
+        if (!isEncrypted(data)) {
```

Comment:
Not sure I really understand what is happening here either.

Reply:
After uploading data, provider.loadContents() is called to refresh the file contents. However, this cannot be applied to encrypted data because they cannot be deserialized into any data type.This check helps determine if we are uploading a plaintext file or an encrypted file. If it's an encrypted file, we skip the refresh to avoid these issues.

Reply:
will make a new encrypted scope store reader to decrypt before load contents